pkgs/by-name/sy/sysdig-cli-scanner/update.sh

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p bash curl coreutils-full jq
```

Comment:
Out of curiosity: which part of update.sh needs coreutils-full?

Reply:
I really put it just in case, I don't actually know if all the commands would work without it. I can remove it if you feel like it's redundant.

Reply:
@tembleking, I'm not too familiar with the resources that update scripts take, so I have no authority on this. That said, I'd go for the smallest possible packages, especially as `coreutils` is used in ~30 update scripts, but `coreutils-full` isn't; at least as far as my quick test shows:

```sh
nix run nixpkgs#silver-searcher -- -0g update.sh | xargs -0 -n1 sed -n 2p | nix run nixpkgs#silver-searcher -- coreutils | nl
     1  #!nix-shell -I nixpkgs=./. -i bash -p jq curl common-updater-scripts nix coreutils
     2  #! nix-shell -i bash -p gnugrep gnused coreutils curl wget jq nix-update prefetch-npm-deps nodejs
     3  #!nix-shell -i bash -p curl jq coreutils common-updater-scripts
# output truncated for demonstration purposes…
```